scripts/bundleSize/getBundleData.test.js

```diff
@@ -0,0 +1,207 @@
+import { getPageBundleData } from './getBundleData';
+
+const pages = [
```

Comment:
This list is already declared twice, should we clean this up a little and define it in a constants file?

Reply:
done